system-test/package.json

```diff
@@ -6,6 +6,7 @@
     "express": "^4.16.3",
     "google-auto-auth": "^0.10.0",
     "got": "^8.0.0",
+    "hard-rejection": "^1.0.0",
```

Comment:
If we have `--throw-deprecation` enabled, you shouldn't need this :)

Reply:
We are running system tests with all of Node 6, 8, and 10, so that doesn't seem to handle all cases yet...

Reply:
I'll land this now, let me know if I need to make a follow-up!